Sources/SwiftInspectorCore/Tests/PropertyAnalzyerSpec.swift

```diff
@@ -0,0 +1,468 @@
+// Created by Tyler Hedrick on 3/27/20.
+//
+// Copyright (c) 2020 Tyler Hedrick
+//
+// Distributed under the MIT License
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+import Nimble
+import Quick
+import Foundation
+
+@testable import SwiftInspectorCore
+
+final class PropertyAnalyzerSpec: QuickSpec {
+  override func spec() {
+    var fileURL: URL!
+    var sut = PropertyAnalyzer(typeName: "FakeType")
+
+    beforeEach {
+      sut = PropertyAnalyzer(typeName: "FakeType")
+    }
+
+    afterEach {
+      guard let fileURL = fileURL else {
+        return
+      }
+      try? Temporary.removeItem(at: fileURL)
+    }
+
+    describe("TypeWithPropInfo.merge(other:)") {
```

Comment:
In this style of testing I think it would be more idiomatic to make a `TypeWithPropInfoSpec` top-level class for testing this related type, as opposed to nesting it within this spec class. But if we do want to keep it here I think it would be good to have two describe invocations here: one for the type name, and one for the method we're testing.